third_party/terraform/data_sources/data_source_sql_backup_run.go

```diff
@@ -0,0 +1,106 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+)
+
+func dataSourceSqlBackupRun() *schema.Resource {
+
+	return &schema.Resource{
+		Read: dataSourceSqlBackupRunRead,
+
+		Schema: map[string]*schema.Schema{
+			"backup_id": {
```

Comment:
Requesting sanity check on using `backup_id` vs `name` for the `id` field in https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/backupRuns#BackupRun

Reply:
Really just for access to the Computed fields of the resource. Definitely not as useful without adding more of the fields to the datasource, but `status` and `start_time` may be useful to a user right now.